packages/common/src/associations/addAssociation.ts

```diff
@@ -1,21 +0,0 @@
-import { IWithAssociations } from "../core/traits/IWithAssociations";
-import { IAssociationInfo } from "./types";
-
-/**
- * Add an association to an entity
- * Persisted into the entity's `.typeKeywords` array
- * @param info
- * @param entity
- */
-export function addAssociation(
```

Comment:
I can deprecate this rather than removing it and have it call `requestAssociation`. `addAssociation` just didn't feel like the correct naming since it's only establishing half of the connection.